service/src/main/java/bio/terra/cbas/dao/RunSetDaoImpl.java

```diff
@@ -0,0 +1,38 @@
+package bio.terra.cbas.dao;
+
+import bio.terra.cbas.models.RunSet;
+import java.util.Optional;
+import java.util.UUID;
+import org.springframework.jdbc.core.namedparam.BeanPropertySqlParameterSource;
+import org.springframework.jdbc.core.namedparam.MapSqlParameterSource;
+import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public class RunSetDaoImpl implements RunSetDao {
+
+  private final NamedParameterJdbcTemplate namedParameterJdbcTemplate;
+
+  public RunSetDaoImpl(NamedParameterJdbcTemplate namedParameterJdbcTemplate) {
+    this.namedParameterJdbcTemplate = namedParameterJdbcTemplate;
+  }
+
+  @Override
+  public int createRunSet(RunSet runSet) {
+    return namedParameterJdbcTemplate.update(
+        "insert into run_set (id, method_id) values (:id, :methodId)",
+        new BeanPropertySqlParameterSource(runSet));
+  }
+
+  @Override
+  public Optional<RunSet> getRunSet(UUID runSetId) {
```

Comment:
I would read this as saying that this method will either return a value, or an empty, and the caller should handle both cases.

If my implementation is throwing an exception just because the row doesn't exist, I'd be inclined to catch that here, rather than force the caller to.